compiler/control/OMROptions.cpp

```diff
@@ -3196,9 +3196,10 @@ const char *OMR::Options::processOptionsAOT(const char *aotOptions, void *feBase
     // Create the default AOT command line options object
     // only do this if this the first time around we're processing options
     //
-    if (!_aotCmdLineOptions)
+    if (!_aotCmdLineOptions) {
         _aotCmdLineOptions = new (PERSISTENT_NEW) TR::Options();
-    else
+        _cmdLineOptions = _aotCmdLineOptions;
+    } else
```

Comment:
AFAIK our coding conventions (which may have changed) require the else part to have brackets, if the 'if' part has brackets.

Reply:
Also, `_cmdLineOptions` is for the JIT options. You are making the JIT options object the same as the AOT options object. It doesn't seem right to me.

Reply:
The original issue mentions a crash but does not describe how/why the crash occurs. Could you please do that either in this PR or in the issue itself?